src/services/LocalizationService.ts

```diff
@@ -31,39 +28,366 @@ class LocalizationService
+                            card4: {
+                                text: "Our service to safely and easily share your code or any other kind of material.",
+                                button: "loremipsum121"
+                            }
+                        },
+                        section5: {
+                            text: "Have you tried turning it off and on again?",
+                            card1: {
+                                text: "I have a didactic problem",
+                                button: "representatives"
+                            },
+                            card2: {
+                                text: "I have a technical problem",
+                                button: "Admins"
+                            }
+                        }
+                    },
+                    aboutUs: {
+                        text1: "We are a non-profit organization, neutral and not affiliated to any political party. Our goal is to provide online services to the students at the University of Milan.",
+                        text2: "You can read our statute here.",
+                        text3: "Here you can find our members and their contacts.",
+                        header1: "Coordinator",
+                        header2: "Administrative committee",
+                        header3: "Administrators and moderators",
+                        discordManager: "loremipsum122"
+                    },
+                    rules: {
+                        text1: "Here you can find the Regulation of our network. Please read it before joining any group.",
+                        question: "Why do we need it?",
+                        answer: "We want to make clear the reasons that took us to introducing these rules into the network. We noticed that most of the groups were flooded with trivial questions re-proposed every day. This was lowering the quality of our chats and discouraged the participation of some students. Therefore, we started repressing this phenomenon warning their proponents and offering a FAQ section for each course.",
+                        text2: "Below you can find the Regulation and its sections. Click on each banner to read the rules.",
+                        text3: "loremipsum123",
+                        section: "loremipsum124",
+                        rules1: {
+                            sectionName: "Questions and FAQs",
+                            rules: []
+                        },
+                        rules2: {
+                            sectionName: "Conduct",
+                            rules: []
+                        },
+                        rules3: {
+                            sectionName: "Materials",
+                            rules: []
+                        },
+                        rules4: {
+                            sectionName: "More",
+                            rules: []
+                        }
```

Comment:
You can add the rules you defined without fields in these arrays of rules. Use any names, like rule1: "", rule2: "", etc.